### PR TITLE
[frontend] Tooltip on group confidence level, displaying overrides

### LINKED ISSUE #6878: Override of confidence level per Entity per Group
## Use case

In 6.1 we introduced the override of confidence per entity at user level, but not at group level. This feature aims to implement it at Group Level.

### PR DESCRIPTION
**Pull request by CelineSebe**
### Proposed changes

* Extraction of Overrides
* Creation of GroupConfidenceLevel component

### Related issues

* #6878 

### Checklist

- [ ] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Review by Archidoit — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/settings/groups/GroupConfidenceLevel.tsx [hunk 1]
```diff
@@ -11,33 +13,39 @@ type GroupConfidenceLevelProps = {
...
-      <Alert severity={'error'} variant={'outlined'}>
-        <AlertTitle>
-          {t_i18n('This group does not have a Max Confidence Level, members might not be able to create data.')}
-        </AlertTitle>
-      </Alert>
+      <Tooltip
+        title={t_i18n('This group does not have a Max Confidence Level, members might not be able to create data.')}
+      >
+        <ReportGmailerrorred fontSize={'small'} color={'error'}/>
+      </Tooltip>
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/settings/groups/GroupConfidenceLevel.tsx]: why this change?

Comment (CelineSebe) [reply opencti-platform/opencti-front/src/private/components/settings/groups/GroupConfidenceLevel.tsx]: Careless mistake, thank you :)

**Review by labo-flg — APPROVED** [review]
